google-cloud-datastore/src/main/java/com/google/cloud/datastore/GqlQuery.java

```diff
@@ -452,6 +452,14 @@ public Map<String, Object> getNamedBindings() {
     return builder.buildOrThrow();
   }
 
+  public Map<String, Binding> getNamedBindingsMap() {
```

Comment:
we might want to consider reducing this to `protected` or marking as InternalApi (same for positional bindings)

Reply:
fixed as part of https://github.com/googleapis/java-datastore/pull/876/commits/28e0332606128204ba3590c9f13007bbbf2cb555.